iron-list.html

```diff
@@ -809,6 +776,7 @@
       if (this.isAttached && !this._itemsRendered && this._isVisible && requiresUpdate) {
         this._lastPage = 0;
         this._update();
+        this._scrollHandler();

```

Comment:
This seems like an actual change... In this branch `_refresh` was basically renamed to `_scrollHandler`, so it's unclear why `_scrollHandler` needed to be added here where there wasn't previously a call to `_refresh`.  Is this addressing something unrelated to integrating the `scrollTarget` behavior?